src/containers/Login/Login.module.scss

```diff
@@ -1,49 +1,49 @@
 @import "../../variables.scss";
-.logoText{
+.logoText {
   font-size: 36px;
-  color: $text-orange ;
+  color: $text-orange;
   font-weight: 500;
 }
 
-.descriptionText{
+.descriptionText {
   font-size: 16px;
   color: $text-grey;
   font-weight: 300;
   margin-top: 20px;
   margin-bottom: 25px;
 }
 
-.logoImage{
+.logoImage {
   width: 100px;
 }
 
-.mt25{
+.mt25 {
   margin-top: 25px;
 }
 
-.text14{
+.text14 {
   font-size: 14px;
 }
 
-.textCenter{
+.textCenter {
   text-align: center;
 }
 
-.textBold{
+.textBold {
   font-weight: bold;
 }
 
-.signInIcon{
+.signInIcon {
   width: 20px;
   height: 20px;
 }
 
-.signInOptionSeperator{
+.signInOptionSeperator {
   position: relative;
 }
 
-.signInOptionSeperator::after{
-  content: '';
+.signInOptionSeperator::after {
+  content: "";
```

Comment:
do this in scss way.
.signInOptionSeperator {
  position: relative;
  &::after{/** your styles*/}
}